File: igor/Problema0224/problema0224.py
arrecado = 0

for x in range(10):
    nome = input("Ecrever o nome: ")
    pagar = float(input("Valor a pagar: "))

    if( pagar >= 250):
        print("Nome cliente: ",nome)
        print("Valor compar: ",pagar)
        print("Desconto: 20%")
        pDescontado = (pagar*80)/100
        print("Valor descontado: ",pDescontado)
        arrecado = arrecado + ((pagar*20)/100)

    else:
        print("Nome cliente: ",nome)
        print("Valor compar: ",pagar)
        print("Desconto: 15%")
        pDescontado = (pagar*85)/100
        print("Valor descontado: ",pDescontado)
        arrecado = arrecado + ((pagar*15)/100)

print("Total arrecado: ",arrecado)
            

File: igor/Problema0224/problema0224.java
import java.util.Scanner;
public class problema0224 {

    static Scanner input = new Scanner(System.in);
    public static void main(String[] args) {

        float arrecado=0;

        for(int i=0;i<10;i++){
            System.out.println("Escrever Nome cliente: ");
            String nome = input.nextLine();
            System.out.println("Valor a pagar: ");
            float pagar = Float.parseFloat(input.nextLine());

            if( pagar >= 250){
                System.out.println("Nome cliente: "+nome);
                System.out.println("Valor compar: "+pagar);
                System.out.println("Desconto: 20%");
                float pDescontado = (pagar*80)/100;
                System.out.println("Valor descontado: "+pDescontado);
                arrecado = arrecado + ((pagar*20)/100);

            }else{
                System.out.println("Nome cliente: "+nome);
                System.out.println("Valor compar: "+pagar);
                System.out.println("Desconto: 15%");
                float pDescontado = (pagar*85)/100;
                System.out.println("Valor descontado: "+pDescontado);
                arrecado = arrecado + ((pagar*15)/100);
            }
            

        }

        System.out.println("Total arrecado: "+arrecado);
    }
}